pkg/yurtctl/cmd/convert/node.go

```diff
@@ -146,18 +146,24 @@ func (c *ConvertNodeOptions) SetupYurthub(workingMode util.WorkingMode) error {
 	if err != nil {
 		return err
 	}
-	kubeletConfPath = strings.Split(kubeletConfPath, "=")[1]
-	apiserverAddr, err := enutil.GetSingleContentFromFile(kubeletConfPath, apiserverAddrRegularExpression)
+	kubeletConfPathArr := strings.Split(kubeletConfPath, "=")
+	if len(kubeletConfPathArr) < 2 {
```

Comment:
I think  const var `kubeletConfigRegularExpression` used in `kubeletConfPath, err := enutil.GetSingleContentFromFile(c.KubeadmConfPath, kubeletConfigRegularExpression)` can guarantee this situation.